locale/es_ES/reader.xml

```diff
@@ -4,19 +4,19 @@
 <!--
   * locale/es_ES/reader.xml
   *
-  * Copyright (c) 2014-2015 Simon Fraser University Library
-  * Copyright (c) 2000-2015 John Willinsky
+  * Copyright (c) 2014 Simon Fraser University Library
+  * Copyright (c) 2000-2014 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
-  *
-  * Localization strings for the es_ES (Español (España)) locale.
-  *
+  * Credits: https://pkp.sfu.ca/wiki/index.php?title=OMP:_Spanish_(es_ES)
+  * Localization strings.
   -->
 
-<locale name="es_ES" full_name = "Español (España)">
+<locale name="es_ES" full_name="Español (España)">
 	<message key="comments.anonymous">Usuario anónimo</message>
-	<message key="comments.anonymousNamed">"{$userName}"</message>
+	<message key="comments.anonymousNamed">&quot;{$userName}&quot;</message>
```

Comment:
The quoting before was consistent with the en_US translation; not sure why quot entities are used here.